QLoggerWriter.cpp

```diff
@@ -50,16 +52,16 @@ QLoggerWriter::QLoggerWriter(const QString &fileDestination, LogLevel level, con
    if (!mFileDestinationFolder.endsWith("/"))
       mFileDestinationFolder.append("/");
 
-   mFileDestination = mFileDestinationFolder + fileDestination;
+   mBareFileDestination = mFileDestinationFolder + fileDestination;
 
    QDir dir(mFileDestinationFolder);
    if (fileDestination.isEmpty())
    {
-      mFileDestination = dir.filePath(QString::fromLatin1("%1.log").arg(
+      mBareFileDestination = dir.filePath(QString::fromLatin1("%1.log").arg(
           QDateTime::currentDateTime().date().toString(QString::fromLatin1("yyyy-MM-dd"))));
    }
    else if (!fileDestination.contains(QLatin1Char('.')))
-      mFileDestination.append(QString::fromLatin1(".log"));
+      mBareFileDestination.append(QString::fromLatin1(".log"));
```

Comment:
Not related with your change, but could you change `fromLatin1` to `fromUtf8`, please?

Reply:
Idem